code/game/objects/items/hot_potato.dm

```diff
@@ -68,17 +68,18 @@
 	return ..()
 
 /obj/item/hot_potato/process()
-	if(stimulant)
-		if(isliving(loc))
-			var/mob/living/L = loc
+	if(isliving(loc))
```

Comment:
Could you do an early return here?
```suggestion
	if(!isliving(loc))
		return
```
Same for the `if(stimulant)` check.
That way we reduce horizontal drift.

Reply:
Done and done, I believe.